_includes/cockroachcloud/download-the-cert-free.md

```diff
@@ -0,0 +1,20 @@
+    <section class="filter-content" markdown="1" data-scope="mac">
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --create-dirs -o ~/.postgresql/root.crt -O https://cockroachlabs.cloud/clusters/<cluster-id>/cert
```

Comment:
If you have a certificate in this directory (e.g., `~/.postgresql/root.crt`), then some drivers (psycopg2, at least) will always fail to verify the SSL authentication for the connection string given by `cockroach demo`.

It might be worth making a note in the `cockroach demo` section of `_includes/vX.X/app/start-cockroachdb.md` about this. At least just a note callout that instructs users to clear all certs from that directory.